src/app/public/modules/radio/radio.component.ts

```diff
@@ -145,9 +196,22 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     return this._value;
   }
 
+/**
+ * Specifies an icon to display in place of the radio button. To group radio buttons like in
+ * [the demo above](https://developer.blackbaud.com/skyux-forms/docs/radio#demo) place the
+ * `sky-switch-icon-group` class on the direct parent element of the radio buttons.
+ */
   @Input()
   public icon: string;
 
+/**
+ * Specifies a type to set the background color after users select a radio button where the icon
```

Comment:
This first sentence is hard to follow, which I think is from when we first wrote the description and the property was new. I'd emulate the wording on the design tab to make this clearer and change the first sentence to `Specifies a type to set the background color after users select an icon radio button.` I think that's much clearer.

Reply:
Looks like this wasn't addressed. Not a huge deal, but just pointing it out so that we can either address it or resolve this comment thread as "will not address."

Reply:
Done